addons/point_of_sale/static/src/js/Screens/PartnerListScreen/PartnerListScreen.js

```diff
@@ -200,8 +200,11 @@ odoo.define('point_of_sale.PartnerListScreen', function(require) {
             let domain = [];
             const limit = 30;
             if(this.state.query) {
-                domain = ['|', ["name", "ilike", this.state.query + "%"],
-                               ["parent_name", "ilike", this.state.query + "%"]];
+                const search_fields = ["name", "parent_name", "phone_mobile_search", "email"];
+                domain = [
+                    ...Array(search_fields.length - 1).fill('|'),
+                    ...search_fields.map(field => [field, "ilike", this.state.query])
```

Comment:
you missed "%". Is it not necessary for the ilike operator?

Reply:
Thanks for your review.
No it's not!

Reply:
I tried and "%" seems to have a purpose. The domain that you are replacing allows search results that pattern-matches '<query>%'. So searching for "fur" will give all with words that starts with "fur".